packages/neo4j-driver-lite/src/index.ts

```diff
@@ -376,7 +379,7 @@ async function hasReachableServer (url: string, config?: Pick<Config, 'logging'>
   }
 }
 
-const USER_AGENT: string = 'neo4j-javascript/' + VERSION
+const BOLT_AGENT = internal.boltAgent.BoltAgent.fromVersion('neo4j-javascript/' + VERSION)
```

Comment:
Maybe move the constant definition before it usage. And I think this line might be `internal.boltAgent.fromVersion`